internal/icinga2/client.go

```diff
@@ -44,8 +43,19 @@ type Client struct {
 	ApiBaseURL       string
 	ApiBasicAuthUser string
 	ApiBasicAuthPass string
+
+	// ApiHttpTransport is a shared http.Transport and http.RoundTripper for all API connections, set by the Launcher.
 	ApiHttpTransport http.RoundTripper
 
+	// ApiTimeout specifies the timeout for API connections outside the Event Stream API.
+	//
+	// Defaults to 1m if unset within Client.Process.
```

Comment:
The way I interpret this is that *it defaults to 1m somewhere else if it's not set in the `Client.Process` method*, you might want to phrase that a little differently, since that doesn't reflect the actual implementation.